src/Sulu/Bundle/ContentBundle/Resources/views/Template/macros/single.html.twig

```diff
@@ -4,7 +4,15 @@
         {{ property.getTitle(userLocale) }}
     </label>
 
-    {% include type.template %}
+    {% if type.template|default(false) %}
```

Comment:
Actually this entire file can be removed quite soon :-) Any reason you've built this if into it?

Reply:
only for the old UI.